docs/components/input.md

```diff
@@ -40,12 +40,7 @@ Use the `labelAttributes` prop to configure the **KLabel's** [props](/components
 <KInput label="Name" :label-attributes="{ help: 'I use the KLabel `help` prop', 'data-testid': 'test' }" />
 
 ```vue
-<KInput
-  label="Name"
-  :label-attributes="{
-    help: 'I use the KLabel `help` prop'
-  }"
-/>
+<KInput label="Name" :label-attributes="{ help: 'I use the KLabel `help` prop' }" />
```

Comment:
This being on multiple lines was throwing off formatting in the `.md` file, so I fixed.